### PR TITLE
merge request test to develop

### PR DESCRIPTION
**Pull request by warrennakamura**
commend of merge @yuknak

### CODE REVIEW

**Comment by warrennakamura** [comment]
reopen @yuknak

**Comment by warrennakamura** [comment]
reopen @yuknak

**Comment by yuknak** [comment]
hmm

**Comment by warrennakamura** [comment]
hmm2

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 #####
 develop
+test
```
Comment (yuknak) [inline README.md:3]: why this?

**Review by yuknak — CHANGES_REQUESTED** [review]
fix this

File: README.md (hunk 1 above)
Comment (warrennakamura) [reply README.md:3]: for test

**Pull request closed without merging**